tools/ci/run_tc.py

```diff
@@ -145,6 +145,27 @@ def install_chrome(channel):
     run(["sudo", "apt-get", "-qqy", "update"])
     run(["sudo", "gdebi", "-qn", "/tmp/%s" % deb_archive])
 
+def install_webkitgtk_from_apt_repository(channel):
+    # Configure webkitgtk.org/debian repository for $channel and pin it with maximum priority
```

Comment:
Please wait for @jgraham to react before changing it, but I wonder if this bit that adds the repo wouldn't make sense to keep in the Dockerfile?

The use of `channel` in webkitgtk.list is the only thing that varies, but I wonder if all available channels could be added, and the `apt-get install` bit uses some qualifier to pick a specific channel?

Reply:
I don't know how to pick a specific channel/suite via command line arguments. I think it can be done via apt pinning (by selecting different priority for a specific channel/suite on the file that is written at `/etc/apt/preferences.d/99webkitgtk`). However, I don't see the benefit of adding all channel/suites to the apt sources list when the bot is instructed to run with a specific channel. Do you see it?

Reply:
The benefit would be that this bit of the setup was a lot more readable as part of the Dockerfile, but if the channel has to be part of the apt sources then it's not a practical suggestion to move it back. I guess the package name is the same for all channels, unlike chrome which has one package/binary name per channel.

Reply:
Indeed. The package name its the same (just different version), its not possible to install both at the same time.

Reply:
Alright, let's stick with this!